### PR TITLE
Add reason for pytest.skipif

### PR DESCRIPTION
**Pull request by mshriver**
When using a boolean as a condition, the reason kwarg is required.

Followup to #8092

### CODE REVIEW

**Review by JacobCallahan — APPROVED** [review]
ACK

**Review by lpramuk — APPROVED** [review]

File: tests/foreman/cli/test_puppetclass.py [hunk 1]
```diff
@@ -37,10 +37,7 @@ def make_puppet():
 
 @pytest.mark.tier2
 @pytest.mark.upgrade
-@pytest.mark.skipif(
-    not settings.repos_hosting_url,
-    reason="repos_hosting_url is not defined in robottelo.properties",
-)
+@pytest.mark.skipif((not settings.repos_hosting_url), reason="Missing repos_hosting_url")
```
Comment (swadeley) [inline tests/foreman/cli/test_puppetclass.py:40]: reason="Missing repos_hosting_url" 
This one has double quotation marks, all the others single.  Does it matter?

**Review by swadeley — COMMENTED** [review]
ACK

File: tests/foreman/cli/test_puppetclass.py (hunk 1 above)
Comment (swadeley) [reply tests/foreman/cli/test_puppetclass.py:40]: oops sorry, I was too late

**Pull request merged**